internal/configuration/request/request.go

```diff
@@ -67,6 +68,32 @@ func initDataRawOption() {
 	}
 }
 
+func initHeaderOption() {
+	cobraParamName := "header"
+	cobraValue := new(customtypes.HeaderSlice)
+	defaultValue := customtypes.StringSlice([]string{})
+	envVar := "PINGCLI_REQUEST_HEADER"
+
+	options.RequestHeaderOption = options.Option{
+		CobraParamName:  cobraParamName,
+		CobraParamValue: cobraValue,
+		DefaultValue:    &defaultValue,
+		EnvVar:          envVar,
```

Comment:
I'm tempted to say no env var as well. Thoughts?

Reply:
makes sense